dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -400,5 +402,33 @@ def principal_approval_url
     def check_idempotency
       Pd::Application::Teacher1819Application.find_by(user: user)
     end
+
+    def meets_criteria
+      response_scores = response_scores_hash
+      scored_questions =
+        if course == 'csd'
+          Teacher1819ApplicationConstants::CRITERIA_SCORE_QUESTIONS_CSD
+        elsif course == 'csp'
+          Teacher1819ApplicationConstants::CRITERIA_SCORE_QUESTIONS_CSP
+        end
+
+      responses = scored_questions.map do |key|
+        response_scores[key]
+      end
+
+      if responses.uniq == [YES]
+        # If all resolve to Yes, applicant meets criteria
+        'Yes'
+      elsif responses.include? NO
+        # If any are No, applicant does not meet criteria
+        'No'
```

Comment:
These strings are the same as the constant values for `YES` and `NO`. Why not use those?